packages/core/src/Card/index.js

```diff
@@ -184,6 +184,7 @@ export default function Card({
       return (
         <Snippet
           fullWidth
+          id={`hurumap-card-${id}`}
```

Comment:
Shouldn't this be handled in content like HURUmap & Flourish: https://github.com/CodeForAfrica/HURUmap-UI/blob/1b8b56af22245b5da748b8d10fda8bf9192784d7/packages/content/src/dataProps.js#L72